seeds/seeds.js

```diff
@@ -1,22 +1,71 @@
-const seedFile = require('knex-seed-file');
+const Promise = require("bluebird");
+
+require("colors");
 
 exports.seed = function(knex, Promise) {
-  return Promise.join(
-    // Deletes ALL existing entries
-    knex('orgs').del(), 
+  console.log("Running seeds.js".yellow)
+  console.log("Deleting all tables".yellow)
+  return knex.raw(
+    `DROP SCHEMA public CASCADE;
+    CREATE SCHEMA public;`
+  ).then(() => {
```

Comment:
I actually think this is not heavy handed in the sense that, each time you run the test, you technically want to build **everything** right? Or is this the wrong way to think about it?